pkg/cluster/internal/create/actions/kubeedge/kubeedge.go

```diff
@@ -177,7 +177,7 @@ func (a *Action) startCloudcoreWithKeadm(ctx *actions.ActionContext, node nodes.
 	// cloudcore svc use NodePort type, to enable edgecore connect to cloudcore, we may add the below routes on the host
 	//iptables -t nat -A PREROUTING -d ${advertise-address} -p tcp --dport 10000 -j DNAT --to-destination ${NODE_IP}:30000
 	//iptables -t nat -A PREROUTING -d ${advertise-address} -p tcp --dport 10002 -j DNAT --to-destination ${NODE_IP}:30002
-	startCmd := fmt.Sprintf("keadm init --advertise-address=%s --profile version=v1.12.0 --kube-config /etc/kubernetes/admin.conf --set cloudCore.hostNetWork=false", a.AdvertiseAddress)
+	startCmd := fmt.Sprintf("keadm init --advertise-address=%s --profile version=v1.17.0 --kube-config /etc/kubernetes/admin.conf --set cloudCore.hostNetWork=false", a.AdvertiseAddress)
```

Comment:
```suggestion
	startCmd := fmt.Sprintf("keadm init --advertise-address=%s --kubeedge-version=v1.17.0 --kube-config /etc/kubernetes/admin.conf --set cloudCore.hostNetWork=false", a.AdvertiseAddress)
```

In version 1.17, you need to use --kubeedge-version to specify the version. You can try it refer to https://github.com/kubeedge/kubeedge/issues/5601